### ISSUE TITLE
Update COMPILING

### ISSUE THREAD

**Issue by beyefendi** [NtQuery/Scylla#42]
COMPILING file says "To compile Scylla you need to have VS2008 installed."
But even the solution file is generated by Visual Studio 2010
Moreover it depends on ATL (<atlbase.h>), hence Visual Studio 2010 Pro needed (namely Express does not compile)

**Comment by greenozon** [comment]
its very easy to switch to VS2010,
do as follows in project properties:  http://prntscr.com/f5mgeh

2) Scylla uses WTL which is based on ATL...

**Comment by LonghronShen** [comment]
Try this VS2017+ branch:
https://github.com/LonghronShen/Scylla

**Comment by NtQuery** [comment]
Scylla was compiled with platform toolset v90 to support Windows XP SP0,1,2,3. That is why you need VS2008 unfortunately. Using a new platform toolset will degrade the support for Windows XP.

**Comment by LonghronShen** [comment]
VS2012+ ships with a Windows XP compatible toolset, e.g., v141_xp.
So I forked to add support for those who want to use new Visual Studio version.

**Comment by NtQuery** [comment]
Yes they have that: https://docs.microsoft.com/en-us/cpp/build/configuring-programs-for-windows-xp?view=vs-2017

But they only support the latest XP Service Pack version.

Maybe it makes sense to finally drop support for older OS versions.

**Comment by LonghronShen** [comment]
You're right, I forgot those who are still using Windows XP SP0,1,2.
Since it is in year 2019 now, we have morden OS, morden building toolset, fantastic git tool, let's make the app more friendly to moden man.

**Comment by greenozon** [comment]
My 5 cents - using Win XP SP0/1 - very dangerous
Marjoriity of SW checks that even if you running dead old WIn XP you must run SP3.
older SP are vulnerable and have lots of  issues.
It does not mean SP3 is secure, but comparing to SP0/1/2 - in an order of magnitude.

2) Regarding XP usage as main OS - well, I know noone out of my many friends who is still using it.
Majority still sits on Win7, but xp - only inside VM and for very very rare use-cases...

I propose to archive/preserve support of Win XP  (as a git tag or branch - whatever) and switch say to W7 / W10 as baseline in master branch

**Comment by NtQuery** [comment]
I think nobody should use XP as main OS, but XP is often used in VMs for reverse engineering, because it is the best OS for that. Mainly due to the fact that it has less security features, it is very easy to go deep into the windows internals + easy to write drivers...
